distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -2825,6 +2836,15 @@ type DB_Table
         # TODO This should ideally be done in a streaming manner, or at least respect the row limits.
         self.read.write path format on_existing_file match_columns on_problems
 
+
+    ## GROUP Standard.Base.Output
+       ICON data_output
+       Creates a Data Link that will act as a view into the query represented by
+       this table.
+    @on_existing_file (Existing_File_Behavior.widget include_backup=False include_append=False)
+    save_as_data_link self destination (on_existing_file:Existing_File_Behavior = Existing_File_Behavior.Error) =
```

Comment:
we need a type on destination (`File`?)

We should have arguments section in documentation.

Reply:
I did not add a type as currently it is only accepting only `Enso_File` but in the future we could allow saving datalinks to other destinations, so I wanted to keep it open ended. 

I guess `Writable_File` should do the trick.

Or I should not be too open ended yet and just do `: Enso_File`. We can always expand the type later as that's a compatible change.